### DISCUSSION TITLE
divide-x with gap doesn`t center the divider line in grid

### DISCUSSION THREAD

**Discussion by teddybee** [tailwindlabs/tailwindcss#5220] [Help]
Good afternoon,

I would like to have few columns in a grid with gap and divider. Unfortunately the line is not in center between the child divs. It is maybe a bug or I miss something. 
I have made a playground of my issue:
[https://play.tailwindcss.com/94ZpAZxegB](https://play.tailwindcss.com/94ZpAZxegB)

**Comment by simonswiss** [comment]
Hey!

This is not a bug (you can't have a border in the middle of a `gap`). If you want to achieve this separator, get rid of the gap and add some padding around each grid cell (children elements):

https://play.tailwindcss.com/qJ7DSom3vp

![image](https://user-images.githubusercontent.com/485747/129661312-71678806-20fc-47de-b151-96b08d729b14.png)

That wouldn't work if you also need vertical padding though, but if you only have three columns like this, that should work fine 👍

**Reply by teddybee** [reply]
Thank you for the response. 
I assumed that would be the proper behaviour if there could be such feature.
The padding in the children will not be 100% good when I want to put the edges on the sides, but I can handle it with negative margin or padding.
PS: I also discovered that, I can't have divider(s) in a multi row grid.
https://play.tailwindcss.com/jZeQpo9BzL
